data/projects/improve_diff_handling.md

```diff
@@ -1,24 +1,41 @@
-Bears should be able to offer more than just one possible patch
-for an issue. Users should be able to choose between different patches, e.g.
-for different possible spelling corrections.
-
-In addition bears should be able to offer templated patches, e.g. to offer
+The advantage of using coala is that bears offer automatically generated diffs
+to fix the problems they detect. This mechanism works smoothly as of now and
+even applies the proposed diff automatically to the problematic file if the
+user so desires. This is very helpful to the users but can be improved by
+giving coala the ability to:
+
+1. Offer multiple diffs i.e. multiple different fixes to the same issue.
+Giving the user a choice to pick the most desireable one.
+2. Incorporate input from the user into the diffs offered by using a
+templating mechanism.
+
+The above improvements can be used in these sample situations. For e.g. when
+suggesting natural language corrections it can offer multiple spelling
+corrections or gramatical corrections.
+
+In addition bears will be able to offer templated patches, e.g. to offer
 variable renamings while the user should be asked for the new variable name.
 
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
 * A [cEP](https://coala.io/cep) for the project has been merged.
 
-##### GSOC 2017 MIDTERM
+##### CODING PHASE 1
 
 * The user is able to choose between multiple patches.
 * The new API is fully documented on <https://api.coala.io/>.
-* At least one bear exists that makes use of this.
+* Proposal and implementation of the bear using multiple patches.
+
+##### CODING PHASE 2
+
+* Implementation of the multiple diff bear is completed and it is merged.
```

Comment:
Is 'multiple diff bear' a new bear to be implemented? I must've not understood this sentence if it's not.